### PR TITLE
Remove usages of '@image' directive

### PR DESCRIPTION
**Pull request by srawlins**
The `{@image` directive does not work, and images can just be specified with the HTML `img` tag. See how the docs currently look:

Screenshot 2023-09-29 at 4 26 00 PM

The MacosSlider image link is broken, so I just removed it.

## Pre-launch Checklist

- [x] I have incremented the package version as appropriate and updated `CHANGELOG.md` with my changes 
- [ ] I have added/updated relevant documentation 
- [ ] I have run "optimize/organize imports" on all changed files
- [x] I have addressed all analyzer warnings as best I could

### CODE REVIEW

**Comment by srawlins** [comment]
Gentle ping; fixing this is required in order to unblock the reporting of unknown doc directives, as this repo is used in Flutter customer tests.

**Comment by goderbauer** [comment]
@GroovinChip could you get this one reviewed to unblock some dartdoc work in Flutter? We are blocked by this because macos_ui has registered its tests in https://github.com/flutter/tests.

**Review by GroovinChip — APPROVED** [review]
LGTM, sorry for the late review

**Pull request merged**
